src/containers/TopicPage/MultidisciplinarySubjectArticle.tsx

```diff
@@ -170,11 +171,13 @@ const MultidisciplinarySubjectArticle = ({ node }: Props) => {
         />
         <HeaderWrapper>
           <HomeBreadcrumb items={breadCrumbs} />
-          <SubjectLinkSet
-            set="test"
-            title={t("multidisciplinarySubject.subjectsLinksDescription")}
-            subjects={subjectLinks ?? []}
-          />
+          {subjectLinks?.length && (
```

Comment:
Legg på `!!` først. Hvis ikke vil denne vise `0` hvis listen er tom

Reply:
Hadde dette blitt fanget av denne?
https://typescript-eslint.io/rules/strict-boolean-expressions/

Reply:
Det hadde ikke sjokkert meg, men vil tro at det er en react-spesifikk regel for dette.